### DISCUSSION TITLE
Eigenvectors are discontinuous

### DISCUSSION THREAD

**Discussion by LiuHaoXuan0403** [ttadano/alamode#102] [Q&A]

**Comment by ttadano** [comment]
Hi,

Is this |**e**(**q**j,k)| of k=Ni and j=1 (first branch) computed along [110]? If yes and if the band crossing occurs along this Brillouin zone path, you need to change j at each **q** point.

The reordering of the branch index can be done automatically by setting `BCONNECT=1` in the &general field, so please try again if this option gives what you expect.
